lib/redmine/menu_manager/top_menu/quick_add_menu.rb

```diff
@@ -52,11 +53,8 @@ def render_quick_add_dropdown
         render(Primer::Beta::Octicon.new(icon: "triangle-down", aria: { label: I18n.t("menus.quick_add.label") }))
       end
 
-      add_first_level_items(menu)
-      if first_level_menu_items_for(:quick_add_menu, @project).present? && work_package_quick_add_items.present?
```

Comment:
❓ You basically omitted the need of those helper methods - which is great. Did you check, if those are still used anywhere else?

I saw in other changes of this PR similar issues, helper functions, that are no longer in use, can you check?